README.md

```diff
@@ -112,7 +112,7 @@ If you would like to see the detailed LICENSE click [here](LICENSE).
 Consult [contributors](https://github.com/oscal-compass/compliance-trestle/graphs/contributors) for a list of authors and [maintainers](MAINTAINERS.md) for the core team.
 
 ```text
-# Copyright (c) 2020 IBM Corp. All rights reserved.
+# Copyright (c) 2024 The OSCAL Compass Authors. All rights reserved.
```

Comment:
What is the story on the headers in the python files? 

Do we need to replace those as well? @jpower432

Reply:
I've seen nothing the in the CNCF read and understand documentation for such a requirement. Just about acceptable licenses, which we have already.

Reply:
Here is some CNCF guidance I found around copyright [notices](https://github.com/cncf/foundation/blob/main/copyright-notices.md). I do think updating existing notices is something we should complete based this guidance and CNCF project convention, but this could be considered out of scope for this PR IMO.

Reply:
The guidance clearly states: `Please note that it is not wrong, and it is acceptable, if a contributor wishes to keep their own copyright notices on their contributions. The above is a recommended format for ease of use, but is not mandated by CNCF.`  My view is that we should leave the existing notices in place, and any new modules will get the new notice. However, it the majority feel uniformity is desirable I will not be a blocker.

Reply:
@degenaro I understand. I did not mean to imply this was a requirement. I read it as a recommendation. In any case, it seems we would need prior approval per this section of the guidance - `You should not change or remove someone else's copyright notice unless they have expressly permitted you to do so. This includes third parties' notices in pre-existing code.`